promtimer/promtimer.py

```diff
@@ -191,7 +193,7 @@ def prepare_grafana(grafana_port, prometheus_base_port, cbcollect_dirs, buckets,
     make_dashboards(data_sources, buckets, times)
 
 def start_grafana(grafana_home_path, grafana_port):
-    log_path = path.join(GRAFANA_DIR, 'grafana.log')
+    log_path = path.join(GRAFANA_LOGS_DIR, 'grafana.log')
```

Comment:
So - what's happening is that are 2 log files.
(1) the grafana log that's specified in custom.ini
(2) the stdout/stderr log

Since they appear to be dupes of each other, we don't need the stdout, stderr log. What we can do is not specify that stdout and stderr should be redirected to a new log. Can you make that change?